### DISCUSSION TITLE
creating activitylogs file on seperate folders like Services

### DISCUSSION THREAD

**Discussion by keraykun** [spatie/laravel-medialibrary#3093] [Q&A]
hello there i am new on SPATIE.  i saw someone code different using spatie log, he seperated the file  folders on SERVICES like in this picture below **App/Services/ActivityLogServices** i want to learn this kind of code. but i don't know how to do it. anyone here know this how?

![wwwwwwwwwwwwwwwwwww](https://user-images.githubusercontent.com/66068632/201451014-c2f26675-fa94-4ace-b1e6-d5656760a012.png)
![image](https://user-images.githubusercontent.com/66068632/201451066-f879a85c-9fb0-4e67-8f74-05e53f347061.png)

**Comment by parallels999** [comment]
This is **medialibrary** nothing to do with logs
